.github/workflows/e2e.yml

```diff
@@ -0,0 +1,55 @@
+name: e2e
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    secrets:
```

Comment:
Why do we need to receive the secret the workflow is called?
When the e2e runs, it will get automatically from the secrets.

Reply:
I set secrets because secrets is not automatically available if workflow_call is used.
But I checked docs again and found `secrets: inherit` options in steps to pass secrets to reusable workflow. Let me fix it in this way. Thanks!